docs/UserGuide.md

```diff
@@ -148,43 +159,49 @@ Expected outcome:
 
 ### Editing a Module: `edit`
 
-Edits the module details of a module that has already been added to GradPad.
-
-To choose the module that you want to edit, specify the index of that module in the Current Modules list.
-You may also choose to edit more than one module detail in the same `edit` command.
+Description:
 
+ * Edits the module details of a module that has already been added to GradPad.
```

Comment:
very minor nit: perhaps "Edit the details of a module...." would be better? 😅

Reply:
Changed the structuring! For the edit {index}, I decided to keep it as {index} for now to match the expected outcome images! I can change it accordingly once we have updated our code

Reply:
noice noted!